tests/components/spokestack/test_tts.py

```diff
@@ -0,0 +1,107 @@
+"""Tests for Spokestack TTS Integration."""
+import asyncio
+import os
+import shutil
+from unittest.mock import patch
+
+from homeassistant.components.media_player.const import (
+    ATTR_MEDIA_CONTENT_ID,
+    DOMAIN as DOMAIN_MP,
+    SERVICE_PLAY_MEDIA,
+)
+import homeassistant.components.tts as tts
+from homeassistant.config import async_process_ha_core_config
+from homeassistant.setup import setup_component
+
+from tests.common import assert_setup_component, get_test_home_assistant, mock_service
+
+
+class TestSpokestackPlatform:
+    """Test Spokestack TTS integration."""
+
+    def setup_method(self):
+        """Set up things to run with start tests."""
+        self.hass = get_test_home_assistant()
+
+        asyncio.run_coroutine_threadsafe(
```

Comment:
We don't write tests like these anymore. Write tests like done for the config flow.

Reply:
For this one, what I understood is that the test should use the `pytest` format and not the older `unittest` like classes?

Reply:
correct. We only write function-based tests. Use pytest.fixtures if you want to run code at startup/shutdown of tests.